components/EntireScheme.svelte

```diff
@@ -18,6 +18,8 @@
 
   gjScheme.update((gj) => {
     gj.authority = authorityName;
+    // we could probably be more sophisticated here and set version more centrally
```

Comment:
You chose well, this is currently the central place to do things like this. It's maybe a bit misleading how this `update` always clobbers things, though.

Reply:
It felt like the right place at this stage but happy to take suggestions on whether we do this elsewhere.